src/features/movies/moviesSaga.js

```diff
@@ -6,13 +6,16 @@ import {
     handleFetchingError,
     finishFetching,
     fetchMovies,
+    setMoviesGenres,
 } from "./moviesSlice";
 
 function* fetchMoviesHandler() {
     yield put(startFetching());
     try {
         const moviesList = yield axios.get("https://api.themoviedb.org/3/movie/popular?api_key=b6338a2fff00b848e44db36dd695b802&language=en-US&page=1");
+        const movieGenres = yield axios.get("https://api.themoviedb.org/3/genre/movie/list?api_key=b6338a2fff00b848e44db36dd695b802&language=en-US");
         yield put(setMoviesList(moviesList));
+        yield put(setMoviesGenres(movieGenres));
```

Comment:
`genres.data.genres.find(...)` in index.js looks really confusing. Maybe we should do moviesGenres.data.genres here. I wonder if we should also try with moviesList.data.results above. I think it is a good topic to talk about on tomorrow daily.